frameworks/projects/Core/src/main/flex/org/apache/flex/utils/Language.as

```diff
@@ -361,5 +370,56 @@ package org.apache.flex.utils
             return boundMethod;
         };
 
+		/**
+		 * @author lizhi
+		 * @param	arr
+		 * @param	names
+		 * @param	opt
+		 */
+		public static function sortOn(arr:Array,names:Object,opt:int=0):void{
```

Comment:
the adobe flash player array.sorton ,opt array.
just use the opt[0],the other [1] [2],not work